client/app/telecom/pack/xdsl/tasks/pack-xdsl-tasks.html

```diff
@@ -0,0 +1,29 @@
+<div class="telecom-pack-xdsl-tasks">
+
+    <div class="telecom-section-content">
+
+        <toast-message></toast-message>
+
+        <div class="widget-presentation">
+            <h2 class="widget-presentation-title"
+                data-translate="xdsl_tasks_list">
+            </h2>
+
+            <oui-datagrid data-rows="XdslTasks.taskIds" data-row-loader="XdslTasks.transformItem($row.id)">
+                <oui-column data-title="::'xdsl_tasks_name' | translate" data-property="function" data-sortable="asc"></oui-column>
+                <oui-column data-title="::'xdsl_tasks_id' | translate" data-property="id" data-sortable></oui-column>
+                <oui-column data-title="::'xdsl_tasks_status' | translate" data-property="status" data-sortable>
+                    <span class="label"
+                          data-ng-class="{
+                                'label-success': $row.status === 'done',
+                                'label-danger': $row.status === 'error' || 'problem',
```

Comment:
This label will always be selected, you meant `$row.status === 'error' || $row.status === 'problem'`, or less lisible but more maintainable `['error', 'problem'].indexOf($row.status) !== -1`